transformers/storage/test_helpers/datatypes.go

```diff
@@ -10,7 +10,8 @@ type BlockMetadata struct {
 
 type VariableRes struct {
 	BlockMetadata
-	Value string
+	DiffID int64 `db:"diff_id"`
```

Comment:
Super minor but maybe worth having a `DiffMetadata` struct that includes `BlockMetadata` and a `DiffID` and then inheriting that instead of both of those in subsequent structs?

Reply:
nice idea, do you mean something like in fdfe915?